src/topBar/TopBar.style.tsx

```diff
@@ -1,47 +1,56 @@
 import styled from 'styled-components'
 
-import { color } from '../_utils/branding'
-
-export const StyledTopBar = styled.header`
-  & {
-    position: relative;
-    width: 100%;
-    background-color: ${color.white};
+import { color, componentSizes, responsiveBreakpoints } from '../_utils/branding'
+
+export const StyledTopBar = styled.header<{ $zIndex: number }>`
+  position: fixed;
+  width: 100%;
+  background-color: ${color.white};
+  display: flex;
+  align-items: center;
+  top: 0;
+  height: ${componentSizes.headerHeight.small};
+  z-index: ${props => props.$zIndex}; /* z-index defined in main application, above drawer */
+
+  .kirk-topBar-inner {
     display: flex;
-    align-items: center;
-  }
-
-  &.kirk-topBar--fixed {
-    position: fixed;
-  }
-
-  &.kirk-topBar--bgTransparent {
-    background-color: transparent;
-  }
-  &.kirk-topBar--bgShadedTransparent {
-    background-color: transparent;
-    background-image: linear-gradient(to bottom, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0));
+    width: 100%;
+    margin: auto;
+    font-weight: 500;
   }
 
-  & .kirk-topBar-left,
-  & .kirk-topBar-right {
+  .kirk-topBar-left,
+  .kirk-topBar-right {
     flex: 1;
     display: flex;
   }
 
-  & .kirk-topBar-left {
+  .kirk-topBar-left {
     justify-content: flex-start;
   }
 
-  & .kirk-topBar-center {
+  .kirk-topBar-center {
     position: absolute;
     top: 50%;
     left: 50%;
     transform: translate(-50%, -50%);
     text-align: center;
   }
 
-  & .kirk-topBar-right {
+  .kirk-topBar-right {
     justify-content: flex-end;
   }
+
+  @media (${responsiveBreakpoints.isMediaLarge}) {
```

Comment:
nit: apply responsive to the item that is changing behavior to better readability